src/Tools/dotnet-monitor/ConfigurationKeys.cs

```diff
@@ -29,5 +29,7 @@ internal static class ConfigurationKeys
         public const string GlobalCounter = nameof(RootOptions.GlobalCounter);
 
         public const string CollectionRuleDefaults = nameof(RootOptions.CollectionRuleDefaults);
+
+        public const string CustomShortcuts = nameof(RootOptions.CustomShortcuts);
```

Comment:
I know that when we've discussed this feature, we've called it "custom shortcuts", but I'm not sure this is the best name for it. First, it's too generic, because by the name, you have no idea what it applies to and when you think of shortcut in general terms, a shortcut is something that takes you from one place to another via a shorter route. Additionally, outside of *.lnk files used as shortcuts to other files on Windows, I'm not sure I've heard "shortcut" used elsewhere in computing.

Consider an alternate name that is more consistent with its usage and context, such as "CollectionRuleTemplates".

Reply:
I would agree, but just for branding consistency - does that mean we should change the name of "Built-In Shortcuts"? That PR is already checked in (but hasn't shipped), so we could do that - unless you only wanted to change the naming of this one.